stream_test.go

```diff
@@ -0,0 +1,103 @@
+package libOpenflow
```

Comment:
This test and benchmark code was used to ensure the code didn't regress somehow. The tests were written to figure out how the code behaves before and after the changes. The benchmarks were used to do the same thing for performance. The behavior of the code and its performance remain the same.